go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/OGKevin/go-bunq
 
-go 1.12
```

Comment:
Sorry that I missed this. You forgot to bump the version in https://github.com/OGKevin/go-bunq/blob/500f3254737cdbb9d6d170e39b1609d00096877c/.travis.yml#L3. Do you mind updating it there as well to ensure that the pipeline is running tests on the correct version.

Reply:
Done!